.github/workflows/release.yml

```diff
@@ -0,0 +1,93 @@
+name: Release OSW-OS Firmware
```

Comment:
Maybe we can merge this new workflow into the existing one, which also builds all firmware variations already?